apps/src/templates/instructions/ScrollButtons.jsx

```diff
@@ -105,39 +116,79 @@ const ScrollButtons = React.createClass({
 
   render() {
 
-    const scrollUpStyle = [
-      styles.arrow,
-      styles.arrowUp,
-      {
-        opacity: this.props.visible ? 1 : 0,
-        top: MARGIN
-      }
-    ];
-
-    const scrollDownStyle = [
-      styles.arrow,
-      styles.arrowDown,
-      {
-        opacity: this.props.visible ? 1 : 0,
-        bottom: -(this.props.height - MARGIN)
-      }
-    ];
+    const upStyle = {
+      opacity: this.props.visible ? 1 : 0,
+      top: this.getMargin()
+    };
+
+    const downStyle = {
+      opacity: this.props.visible ? 1 : 0,
+      bottom: -(this.props.height - this.getMargin())
+    };
+
+    // for most tutorials, we have minimalist arrow elements. For
+    // minecraft, we use a special button element to stylistically align
+    // with the other buttons on the screen.
+
+    const upButton = (this.props.isMinecraft) ?
```

Comment:
yeah, I spend far too long trying to find a good way to do this.

I think the best longer-term solution is to redesign our existing scroll buttons to be actual buttons, instead of divs with way-too-fancy CSS